app/V1Module/presenters/UsersPresenter.php

```diff
@@ -548,4 +548,25 @@ public function actionInvalidateTokens(string $id) {
     ]);
   }
 
+  public function checkSetAllowed(string $id) {
+    $user = $this->users->findOrThrow($id);
+    if (!$this->userAcl->canSetIsAllowed($user)) {
+      throw new ForbiddenRequestException();
+    }
+  }
+
+  /**
+   * Set "isAllowed" flag of the given user. The flag determines whether a user may perform any operation of the API.
+   * @POST
+   * @param string $id Identifier of the user
+   * @Param(type="post", name="isAllowed", validation="bool", description="Whether the user is allowed (active) or not.")
+   * @throws InvalidArgumentException
+   * @throws NotFoundException
+   */
+  public function actionSetAllowed(string $id) {
+    $user = $this->users->findOrThrow($id);
+    $user->setIsAllowed((bool)$this->getRequest()->getPost("isAllowed"));
```

Comment:
I am not really sure if this will work... Did you check it and performed real HTTP request against this endpoint?

Reply:
No, it is tested by unit test only.

Reply:
Well if I am not mistaken, the parameter from the post will always be string... therefore conversion to boolean of `false` will result in `true`.